docroot/config/sync/core.entity_form_display.paragraph.map_pagagraph.default.yml

```diff
@@ -0,0 +1,24 @@
+uuid: a960ed46-1cee-4181-b354-8d4c75c1ab50
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.map_pagagraph.field_map
+    - paragraphs.paragraphs_type.map_pagagraph
```

Comment:
Please rename to google map paragraph because we will have maps defined by mapael.